lib/app_web/templates/page/index.html.eex

```diff
@@ -1,9 +1,40 @@
 <section class="tc">
-  <h2>Empower people to maximise effectiveness, creativity and happiness</h2>
+	<img src="https://dwyl.com/img/common/dwyl-heart-only-logo.png" width="100px" class="center h-auto db" />
+	<span class="dwyl-teal f2">dwyl</span>
+  <p class="f5"><strong>Unlock</strong> a new level of <strong>personal effectiveness</strong>.</p>
 </section>
 
 <section class="tc">
-  <%=link("Capture", to: Routes.capture_path(@conn, :new), class: "f6 b link dim br2 ph3 pv2 db white bg-dark-blue center w4 ma4") %>
+<!-- we don't need buttons to perform these actions the flow should be effortless.
+  <%=link("Capture", to: Routes.capture_path(@conn, :new),
+  class: "f6 b link dim br2 ph3 pv2 db white bg-dark-blue center w4 ma4") %>
   <%=link("Categorise", to: Routes.categorise_path(@conn, :index), class: "f6 b link dim br2 ph3 pv2 db white bg-dark-blue center w4 ma4") %>
   <%=link("Complete", to: "/", class: "f6 b link dim br2 ph3 pv2 db white bg-dark-blue center w4 ma4") %>
+-->
+	<a href="<%= @url_oauth_google %>"> <!-- Google Auth Blue Button 8kb - we can optimise this past-MVP! -->
+		<img src="https://user-images.githubusercontent.com/194400/69724732-62090d80-1114-11ea-8d81-7ef4fb179f3c.png"
+		alt="Sign in with Google" />
+	</a>
+
+</section>
+
+<span class="b db tc pa2">OR</span>
+<section class="tc">
+	<%= form_for @changeset, Routes.page_path(@conn, :register), fn f -> %>
+	<div class="pa2">
+		<%= text_input f, :email, placeholder: "email" %>
```

Comment:
![image](https://user-images.githubusercontent.com/194400/70631070-82f95480-1c24-11ea-8a46-0ff1732bde8b.png)

Should be `<input type="email" ... >` to allow browser to auto-complete email address.
https://hexdocs.pm/phoenix_html/Phoenix.HTML.Form.html#email_input/3